gazebo_plugins/test/bumper_test/test_bumper.py

```diff
@@ -87,12 +87,12 @@ def checkContact(self):
         # see if total wrench is close to 98.1N
         if self.sample_count > 20:
           if abs(self.fz_avg - 98.1) < 0.01:
-            print "z force ",self.fz_avg
+            print("z force ", self.fz_avg)
             self.success = True
 
 
     def test_bumper(self):
-        print "LNK\n"
+        print("LNK")
```

Comment:
Is the `\n` removed on purpose?

Reply:
https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1069/commits/d4198f56f4cd8510139aa36ff5693bc97751d21e